### PR TITLE
Fix Typo

### PR DESCRIPTION
**Pull request by kandros**

### CODE REVIEW

**Comment by merelinguist** [comment]
Could you also change `where as` to `whereas`?

**Comment by kandros** [comment]
> Could you also change `where as` to `whereas`?

grammarly.com to the rescue :D

File: packages/server/src/synchronizer/README.md [hunk 1]
```diff
@@ -5,18 +5,18 @@ Design goals
...
 - Simplify tests
 
 # Why Streams?
 
-Initially blitz will be used by people with small projects however as the number files and throughput increases we will need to use an architecture that allows for large paralell throughput with low memory consumption. Node is built on streams as a primitive so it makes sense to utilize what is available. The Gulp ecosystems provides several tools to managing streams of files so that makes sense to use those tools when available. Because refactoring to streams later would be extremely difficult and painful not starting with streams would be a design mistake.
+Initially, blitz will be used by people with small projects however as the number files and throughput increases we will need to use an architecture that allows for large parallel throughput with low memory consumption. Node is built on streams as a primitive so it makes sense to utilize what is available. The Gulp ecosystems provide several tools for managing streams of files so that makes sense to use those tools when available. Because refactoring to streams later would be extremely difficult and painful not starting with streams would be a design mistake.
```
Comment (merelinguist) [inline packages/server/src/synchronizer/README.md:15]: Please capitalise all instances of “Blitz”

**Comment by merelinguist** [comment]
Thank you!

**Pull request merged**